meson_options.txt

```diff
@@ -1,9 +1,9 @@
 option('blacklist_file', type : 'string', value : 'appcenter.blacklist', description : 'The name of the file listing blacklisted applications')
```

Comment:
I appreciate that you're adding a new option here so you have to modify this file. But if you weren't, would it not make more sense to set the option values in `debian/rules` using debhelper rather than overriding the defaults. You'd be less likely to bump into conflicts that way since you're adding all the debian files to the master branch yourselves anyway.